jenkins-integration/beaker/install/shared/install_deps.rb

```diff
@@ -0,0 +1,9 @@
+step "Configure epel" do
+  tmp_module_dir = master.tmpdir('configure_epel')
+  on(master, puppet('module', 'install', 'stahnma-epel', '--codedir', tmp_module_dir))
+  on(master, puppet('apply', '-e', "'include epel'"))
```

Comment:
Oh, hm.  Yeah, not sure how I missed that.  Adding the "--codedir tmp_module_dir" would seem to make sense here - will do that.

I was using a tmp_module_dir mostly because I didn't want this to go into the normal production code dir that might be on the SUT - potentially conflicting with the code deployment that the specific test might do.  Does that sound reasonable?

I'd be okay with nuking the tmp_module_dir on completion.  Maybe I could just have it nuke the directory if the step was successful?  Might be useful to keep it around in the case of a failure for troubleshooting after the fact.